### ISSUE TITLE
preds file has many meaningless tokens.

### ISSUE THREAD

**Issue by 27182812** [rpryzant/delete_retrieve_generate#22]
I tried my own data which similar to positive and negative data. Though epochs I trained are not many，but the result is extremely bad，with lots of meaningless tokens. Can you help me? Thank you very much!
![image](https://user-images.githubusercontent.com/33630730/112944080-eafae380-9164-11eb-928e-2d2a79e70f0b.png)

**Comment by 27182812** [comment]
What confuses me most is that the original experiment, such as yelp, is basically complete sentences in pred1.

**Comment by rpryzant** [comment]
Hmm how large are your data, how many epochs are you training the model for, and at what learning rate? It seems like the model hasn't converged. I would recommend trying out different combinations of hyperparameters to achieve a better fit.

**Comment by 27182812** [comment]
Thank you very much! I solved it.

**Comment by wasedaward** [comment]
> Thank you very much! I solved it.

Sorry for interrupting you. Can I ask about your combinations of hyperparameters? Because I met the same trouble. Thank you in advance.
